SeniorInterview/StocksAnalyzed/src/Main.java

```diff
@@ -22,5 +23,14 @@ public static void main(String[] args) {
             if (count > 5) break;
             count++;
         }
+
+        System.out.println("\nTOP 3 past variation");
+        System.out.println("==============================");
+
+        // Variacao de alguns dias passados
```

Comment:
Would be nice to add more details to the comment - num of days for the variation e.g.